extensions/dicom-sr/src/getSopClassHandlerModule.js

```diff
@@ -26,12 +26,16 @@ const CodeNameCodeSequenceValues = {
   ImageLibraryGroup: '126200',
   TrackingUniqueIdentifier: '112040',
   TrackingIdentifier: '112039',
+  Finding: '121071',
 };
 
 const RELATIONSHIP_TYPE = {
   INFERRED_FROM: 'INFERRED FROM',
 };
 
+const CORNERSTONE_CODING_SCHEME_DESIGNATOR = 'CST4';
+const CORNERSTONE_FREETEXT_CODE_VALUE = 'CORNERSTONEFREETEXT';
```

Comment:
Almost need this in `@ohif/core` or a separate library that manages/describes our custom definitions/magic

Reply:
Yeah we need all this doc'd for `dcmjs`.